hw/xbox/nv2a.h

```diff
@@ -20,6 +20,84 @@
 #ifndef HW_NV2A_H
 #define HW_NV2A_H
 
+// #define DEBUG_NV2A
```

Comment:
put this into nv2a_debug.{c,h}

that or put it in nv2a_int.h and mark gl_* as inline